Unity-Technologies/ScriptableRenderPipeline
# [Backport] FB#1129900 - Invalid dependency paths when creating Unlit graphs

### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

Backporting FB#1129900 - Invalid dependency paths when creating Unlit graphs PR #3135 .

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
All Tests Green
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=lwrp%2Fbugfix-1129900-backport&unity_branch=2019.1%2Fstaging&automation-tools_branch=master

**Manual Tests**: What did you do?
- Other: 
Hand-tested creating Unlit SHaders in LWRP to confirm exception is gone. 

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High? None

**Halo Effect**: None, Low, Medium, High? None

---
### Comments to reviewers
Notes for the reviewers you have assigned.
